app/containers/Chip/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Pressable, StyleSheet, View, Text, StyleProp, TextStyle, ViewStyle } from 'react-native';
+
+import { ISelectedUser } from '../../reducers/selectedUsers';
+import { useTheme } from '../../theme';
+import { CustomIcon, TIconsName } from '../CustomIcon';
+import sharedStyles from '../../views/Styles';
+import Avatar from '../Avatar';
+
+const styles = StyleSheet.create({
+	pressable: {
+		marginVertical: 16,
+		paddingHorizontal: 8,
+		marginRight: 8,
+		borderRadius: 2,
+		height: 40,
```

Comment:
What happens if the user has a larger text setting on their device?
Your accessibility breaks.
![image](https://user-images.githubusercontent.com/804994/183984975-dcba8378-5750-42d5-a4ce-7609af240d20.png)

We should try not to set a fixed height for components like this and use padding instead.